### PR TITLE
[Jest] Migrate min build tests

### PR DESCRIPTION
**Pull request by birkskyum**
Issue with global object in last test.

### CODE REVIEW

**Comment by birkskyum** [comment]
I have left out the odd browserify test, because it made a world of trouble with Jest, and the world is quickly moving towards es modules anyway.

File: test/build/min.test.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import packageJson from '../../package.json';
+import browserify from 'browserify';
+// import {fileURLToPath} from 'url';
+
+// const __dirname = dirname(fileURLToPath(import.meta.url));
+const minBundle = fs.readFileSync('dist/maplibre-gl.js', 'utf8');
+
+describe('test min build', () => {
+    test('production build removes asserts', done => {
+        expect(minBundle.indexOf('canary assert') === -1).toBeTruthy();
```
Comment (HarelM) [inline test/build/min.test.ts:13]: I'm sure this expect can be prettier - using `contains`?

File: test/build/min.test.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        done();
+    });
+
+    test('trims package.json assets', done => {
+    // confirm that the entire package.json isn't present by asserting
+    // the absence of each of our script strings
+        for (const name in packageJson.scripts) {
+            expect(minBundle.indexOf(packageJson.scripts[name]) === -1).toBeTruthy();
+        }
+        done();
```
Comment (HarelM) [inline test/build/min.test.ts:24]: No need for `done` I think

File: test/build/min.test.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    });
+
+    test('trims reference.json fields', done => {
+        expect(reference.$root.version.doc).toBeTruthy();
+        expect(minBundle.indexOf(reference.$root.version.doc) === -1).toBeTruthy();
+        done();
```
Comment (HarelM) [inline test/build/min.test.ts:30]: Same for this `done`

**Review by HarelM — CHANGES_REQUESTED** [review]

**Comment by birkskyum** [comment]
I think the codemods script added done() to everything because the last test is async. Anyway, it's gone now, and I added the .includes() instead of the indexOf() === -1

**Review by HarelM — APPROVED** [review]

**Pull request merged**